Formula/libb64.rb

```diff
@@ -0,0 +1,20 @@
+class Libb64 < Formula
+  desc "Base64 encoding/decoding library"
+  homepage "https://libb64.sourceforge.io/"
+  url "https://downloads.sourceforge.net/project/libb64/libb64/libb64/libb64-1.2.src.zip"
```

Comment:
Looks like a newer version: https://sourceforge.net/projects/libb64/files/libb64/libb64/libb64-1.2.1.zip/download

This hasn't been updated for a very long time but it's packaged by Debian (https://packages.debian.org/source/stretch/libb64)

Reply:
@schloegl Can you please update to the most recent version?